android/src/main/java/com/thebluealliance/androidclient/helpers/StatsHelper.java

```diff
@@ -0,0 +1,17 @@
+package com.thebluealliance.androidclient.helpers;
+
+import java.text.DecimalFormat;
+
+public final class StatsHelper {
```

Comment:
Do this in [`ThreadSafeFormatters`](https://github.com/the-blue-alliance/the-blue-alliance-android/blob/master/android/src/main/java/com/thebluealliance/androidclient/helpers/ThreadSafeFormatters.java) instead, so we can avoid race conditions